src/axom/quest/examples/scattered_interpolation.cpp

```diff
@@ -444,7 +444,7 @@ struct Input
   int dimension {2};
   std::vector<double> boundsMin;
   std::vector<double> boundsMax;
-  std::string outputProtocol = SIDRE_DEFAULT_PROTOCOL;
+  std::string outputProtocol = sidre::Group::getDefaultIOProtocol();
 
   const std::set<std::string> s_validProtocols {"json",
```

Comment:
Any chance that `sidre::Group::getDefaultIOProtocol()` will return something not in `s_validProtocols`?  Will it matter even if it did?

Reply:
I see that Group.hpp around line 45 defines `SIDRE_DEFAULT_PROTOCOL` to be `"sidre_hdf5"` if we have HDF5, `"sidre_conduit_json"` if we don't.  Should `"sidre_conduit_json"` be a member of `s_validProtocols` (or again, does it matter)?

Reply:
I don't know this example, so I tried not to change it beyond getting rid of the macro.